### ISSUE TITLE
Changing Soundfont Softlocks Game 

### ISSUE THREAD

**Issue by PillowFortMaster** [Fasguy/BaldisBasicsModMenu-archive#78]
**What's the Version of the game you're using?**

V1.1

**What's the Version of the mod you're using?**

V1.1.0

**What Operating System are you using?**
Please provide the full Operating System name (example: Windows 10 Pro 64 Bit)

Windows 11 Insiders Build 64 Bit Home

**Describe the bug**

I downloaded a realistic soundfont and put it into the game and for some reason it instantly softlocked the game and crashed.

**Expected behavior**

Change the soundfont

**Comment by Fasguy** [comment]
I'm at the mercy of what MPTK actually supports and i've noticed that some soundfonts just fail to load without any response. A full crash i have not seen yet myself.
Please upload your Player.log file here, so i can investigate this. (`C:\Users\<your Windows user>\AppData\LocalLow\Basically Games\Baldi's Basics Classic Remastered\Player.log`)

**Comment by PillowFortMaster** [comment]
[Player.log](https://github.com/Fasguy/BaldisBasicsModMenu-archive/files/13501581/Player.log)

**Comment by Fasguy** [comment]
Yep, as expected, there isn't much useful information beyond MPTK trying to load the soundfont. This appears to be either an issue with MPTK itself or the soundfont, rather than the modifiers, so there isn't really much i can do here.
